src/Microsoft.PowerShell.Commands.Utility/commands/utility/FormatAndOutput/format-hex/Format-Hex.cs

```diff
@@ -23,6 +23,23 @@ public sealed class FormatHex : PSCmdlet
     {
```

Comment:
@vexx32, your last commit had 1 failures in `PowerShell-CI-linux`
Clean up open Runspaces when exit powershell process.PowerShell process should not freeze at exit
```powershell
Expected $true, but got $false.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/engine/Api/BasicEngine.Tests.ps1: line 53
53:         Wait-UntilTrue -sb { $process.HasExited } -TimeoutInMilliseconds 5000 -IntervalInMilliseconds 1000 | Should -BeTrue
```

Reply:
🤔 This one's getting a bit too common. Do we have a tracking issue for these @iSazonov?